test/unit_tests/braket/aws/test_aws_quantum_task.py

```diff
@@ -414,53 +415,232 @@ def test_from_circuit_with_shots_value_error(aws_session, arn, circuit):
+                        "readoutThermalizationDuration": 256,
+                        "reduceIntersampleCorrelation": False,
+                        "reinitializeState": True,
+                        "resultFormat": "RAW",
+                        "spinReversalTransformCount": 100,
+                    }
+                },
             ),
             "arn:aws:braket:::device/qpu/d-wave/Advantage_system1",
         ),
         (
             DwaveAdvantageDeviceParameters.parse_obj(
-                {"deviceLevelParameters": {"autoScale": "False"}}
+                {
+                    "deviceLevelParameters": {
+                        "annealingOffsets": [3.67, 6.123],
+                        "annealingSchedule": [[13.37, 10.08], [3.14, 1.618]],
+                        "annealingDuration": 1,
+                        "autoScale": False,
+                        "beta": 0.2,
+                        "chains": [[0, 1, 5], [6]],
+                        "compensateFluxDrift": False,
+                        "fluxBiases": [1.1, 2.2, 3.3, 4.4],
+                        "initialState": [1, 3, 0, 1],
+                        "maxResults": 1,
+                        "programmingThermalizationDuration": 625,
+                        "readoutThermalizationDuration": 256,
+                        "reduceIntersampleCorrelation": False,
+                        "reinitializeState": True,
+                        "resultFormat": "RAW",
+                        "spinReversalTransformCount": 100,
+                    }
+                },
             ),
             "arn:aws:braket:::device/qpu/d-wave/Advantage_system1",
         ),
         (
             Dwave2000QDeviceParameters.parse_obj(
-                {"deviceLevelParameters": {"postprocessingType": "OPTIMIZATION"}}
+                {
+                    "deviceLevelParameters": {
+                        "postprocessingType": "OPTIMIZATION",
+                        "annealingOffsets": [3.67, 6.123],
+                        "annealingSchedule": [[13.37, 10.08], [3.14, 1.618]],
+                        "annealingDuration": 1,
+                        "autoScale": False,
+                        "beta": 0.2,
+                        "chains": [[0, 1, 5], [6]],
+                        "compensateFluxDrift": False,
+                        "fluxBiases": [1.1, 2.2, 3.3, 4.4],
+                        "initialState": [1, 3, 0, 1],
+                        "maxResults": 1,
+                        "programmingThermalizationDuration": 625,
+                        "readoutThermalizationDuration": 256,
+                        "reduceIntersampleCorrelation": False,
+                        "reinitializeState": True,
+                        "resultFormat": "RAW",
+                        "spinReversalTransformCount": 100,
+                    }
+                }
             ),
             "arn:aws:braket:::device/qpu/d-wave/DW_2000Q_6",
         ),
```

Comment:
```suggestion
        ),
        (
            Dwave2000QDeviceParameters.parse_obj(
                {"deviceLevelParameters": {}}
            ),
            "arn:aws:braket:::device/qpu/d-wave/DW_2000Q_6",
        ),
```

Wouldn't hurt to add a test for empty/incomplete device level parameters